src/Microsoft.AspNetCore.SignalR.Core/DefaultHubLifetimeManager.cs

```diff
@@ -9,7 +9,7 @@
 
 namespace Microsoft.AspNetCore.SignalR
 {
-    public class DefaultHubLifetimeManager<THub> : HubLifetimeManager<THub>
+    public class DefaultHubLifetimeManager<THub> : HubLifetimeManager<THub> where THub : Hub
```

Comment:
I noticed there were some classes with generic arguments for a hub but had no type constraint. I've added them but was it intentional? If so then let me know and I'll remove and add comments with a reason

Reply:
That's good, we just forgot them.